pkg/virt-launcher/virtwrap/agent-poller/agent_poller.go

```diff
@@ -192,7 +192,7 @@ type PollerWorker struct {
 type agentCommandsExecutor func(commands []AgentCommand)
 
 // Poll is the call to the guestagent
-func (p *PollerWorker) Poll(execAgentCommands agentCommandsExecutor, closeChan chan struct{}) {
+func (p *PollerWorker) Poll(execAgentCommands agentCommandsExecutor, closeChan chan struct{}, pollInitialInterval time.Duration) {
```

Comment:
We should try not to shadow other object names. This one shadows the const.
How about `initialInterval`? The fact that it is an argument of `Poll` allows us to remove this prefix.

Please also change the unit-test that calls it, otherwise this is confusing (I prefer the commit to compile, showing all areas that it changes)